libs/dal/src/repositories/integration/integration.entity.ts

```diff
@@ -1,6 +1,10 @@
 import { ChannelTypeEnum } from '@novu/shared';
 
 export interface ICredentials {
+  authKey?: string;
+  sender?: string;
```

Comment:
If we use the 'from' we can remove this one here.

Reply:
Ok we can do this I will manage it accordingly.

Reply:
This hasn't been addressed yet.

Reply:
@p-fernandez resolved and pushed

Reply:
This hasn't been resolved. The property `sender` is still appearing in the interface.

Reply:
![image](https://user-images.githubusercontent.com/94128805/208411836-69b6a605-bbac-467a-bf43-6afb9d1d86c0.png)
Hi Team @p-fernandez @scopsy  I have already pushed the codes but I don't know why it is not showing to you can you please check whether it is showing or not and if it is not showing please tell me I will have a look it again.